frontend/src/libs/maps/icon-name-to-icon.map.ts

```diff
@@ -0,0 +1,25 @@
+import { ReactComponent as ArrowIcon } from '#assets/icons/arrow.svg';
+import { ReactComponent as BackwardIcon } from '#assets/icons/backward.svg';
+import { ReactComponent as ForwardIcon } from '#assets/icons/forward.svg';
+import { ReactComponent as NextIcon } from '#assets/icons/next.svg';
+import { ReactComponent as PauseIcon } from '#assets/icons/pause.svg';
+import { ReactComponent as PlayIcon } from '#assets/icons/play.svg';
+import { ReactComponent as PlusIcon } from '#assets/icons/plus.svg';
+import { ReactComponent as PreviousIcon } from '#assets/icons/previous.svg';
+import { type IconName } from '#libs/types/types.js';
+
+const iconNameToIcon: Record<
```

Comment:
I suppose we already have this mapper into development but in different location

Reply:
As I see this mapper exists in src/libs/maps/icon-name-to-plain-svg.ts in development as mine file. I just renamed this file, because the name of the file should be the same as the exported function inside it. So we have iconNameToIcon, but a file named iconNameToPlainSvg in development That's why I renamed it and added maps.ts for consistency.